recipe/meta.yaml

```diff
@@ -39,13 +39,14 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - armpl  # [aarch64]        
+        - armpl  # [aarch64]
         - ninja-base
         - pkg-config
       host:
         - python
         - pip
         - cython >=0.29.34,<3.1
+        - meson-python >=0.15.0,<0.16.0
         - python-build
         # dependencies of vendored meson-python
```

Comment:
all dependencies of vendored meson-python can be removed:
https://github.com/numpy/numpy/compare/v1.26.3...v1.26.4#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711L7

Reply:
Done